src/test/kotlin/com/geekbeast/rhizome/jobs/EmptyJobState.kt

```diff
@@ -0,0 +1,47 @@
+package com.geekbeast.rhizome.jobs
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import org.apache.commons.lang3.RandomUtils
+import java.lang.IllegalStateException
+import java.util.*
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+data class EmptyJobState(val name: String) : JobState
+
+class EmptyJob(
+        state: EmptyJobState,
+        fail: Boolean = false,
+        private val rounds: Int = 10
+) : AbstractDistributedJob<Long, EmptyJobState>(state) {
+    private var round = 0
+    var fail: Boolean = fail
+        private set
+
+    @JsonCreator
+    constructor(
+            id: UUID?,
+            taskId: Long?,
+            status: JobStatus,
+            progress: Byte,
+            hasWorkRemaining: Boolean,
+            state: EmptyJobState,
+            result: Long?,
+            fail: Boolean
+    ) : this(state) {
+        initialize(id, taskId, status, progress, hasWorkRemaining,result)
+        this.result = result
+        this.fail = fail
+    }
+
+    override fun processNextBatch() {
+        Thread.sleep(RandomUtils.nextLong(100, 500))
+        result = 5
```

Comment:
what's the purpose of this?

Reply:
(setting result = 5, I mean)

Reply:
To have an expected value to test against.